### PR TITLE
Feature demo

### PR DESCRIPTION
**Pull request by RamonGilabert**
Not ready yet as it needs the method from the pod, but it's the initial skeleton of it.

@vadymmarkov, @zenangst

### CODE REVIEW

File: Demo/ImaginaryDemo/ImaginaryDemo/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    static let imageNumber = 20
+  }
+
+  lazy var imaginaryArray: [NSURL] = {
+    let faker = Faker()
+    var array = [NSURL]()
+
+    for i in 0..<Constants.imageNumber {
+      if let imageURL = NSURL(
+        string: "http://lorempixel.com/\(Constants.imageWidth)/\(Constants.imageHeight)/?type=attachment&id=\(i)\(50)") {
```
Comment (vadymmarkov) [inline Demo/ImaginaryDemo/ImaginaryDemo/ViewController.swift]: You can use faker.internet.image(Constants.imageWidth, Constants.imageHeight) + "?type=attachment&id=(i)(50)"

Comment (RamonGilabert) [inline Demo/ImaginaryDemo/ImaginaryDemo/ViewController.swift]: Is this new?

Comment (RamonGilabert) [inline Demo/ImaginaryDemo/ImaginaryDemo/ViewController.swift]: Done!

**Comment by vadymmarkov** [comment]
@RamonGilabert :star2:

**Pull request merged**